base-template/README.md

```diff
@@ -0,0 +1,28 @@
+# What's in the base-template directory
+
+This base-template directory contains templates and guides to be followed by a `doctype` template author. They can also be used by a project's docset owner select and customize the project's templates.
```

Comment:
"docset owner select" should be changed to "docset owner to select"

Reply:
Thanks, fixed.